src/Randomizer.App/BoolToVisibilityConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Randomizer.App;
+
+[ValueConversion(typeof(bool), typeof(Visibility))]
+public class BoolToVisibilityConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return (bool) value ? Visibility.Visible : Visibility.Collapsed;
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter,CultureInfo culture)
+    {
+        throw new NotImplementedException();
```

Comment:
Guessing we never plan to convert back to a `bool`?

Reply:
I don't think the other way would make sense, really. I'm not sure what two way binding you could do would change between the enum visibility values and flow well from a user perspective.

I just use this as an easy way to hide stuff based on booleans.

Reply:
Gotcha, makes sense to me!